[2022-12-16T08:08:38.909+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop scheduled__2022-12-15T00:00:00+00:00 [queued]>
[2022-12-16T08:08:38.914+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop scheduled__2022-12-15T00:00:00+00:00 [queued]>
[2022-12-16T08:08:38.914+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-16T08:08:38.915+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-16T08:08:38.915+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-16T08:08:38.954+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): run_after_loop> on 2022-12-15 00:00:00+00:00
[2022-12-16T08:08:38.959+0000] {standard_task_runner.py:55} INFO - Started process 46578 to run task
[2022-12-16T08:08:38.965+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'scheduled__2022-12-15T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', '/home/azureuser/.local/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpmt3_auc3']
[2022-12-16T08:08:38.967+0000] {standard_task_runner.py:83} INFO - Job 14: Subtask run_after_loop
[2022-12-16T08:08:39.029+0000] {task_command.py:389} INFO - Running <TaskInstance: example_bash_operator.run_after_loop scheduled__2022-12-15T00:00:00+00:00 [running]> on host vibrdairflow.tciw5gv3lhjetcyg0kqzfvdylc.hx.internal.cloudapp.net
[2022-12-16T08:08:39.103+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2022-12-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-15T00:00:00+00:00
[2022-12-16T08:08:39.104+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-16T08:08:39.104+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo 1']
[2022-12-16T08:08:39.112+0000] {subprocess.py:86} INFO - Output:
[2022-12-16T08:08:39.113+0000] {subprocess.py:93} INFO - 1
[2022-12-16T08:08:39.113+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-16T08:08:39.164+0000] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20221215T000000, start_date=20221216T080838, end_date=20221216T080839
[2022-12-16T08:08:39.217+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-16T08:08:39.230+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
